Changed program Structure

r-stha opened the pull request:
Made a new file for each class.
Made a new file and class for accepting choices from a option.

hstha commented on ArthmeticCalculation.cs:
```diff
@@ -0,0 +1,85 @@
+    {
+
+        Console.WriteLine("Enter the base:");
+        double Base = Convert.ToDouble(Console.ReadLine());
```
variable name never start with uppercase

r-stha replied:
named as radix

hstha commented on ArthmeticCalculation.cs:
```diff
@@ -0,0 +1,85 @@
+
+    public static void Subtraction()
+    {
+        var (operand1, operand2) = Input.TakeArthmeticInput();
```
Since you are learning, never use var, be specific on which datatype are you using

r-stha replied:
given datatype of double to both variable

hstha commented on ArthmeticCalculation.cs:
```diff
@@ -0,0 +1,85 @@
+    public static void Division()
+    {
+        var (operand1, operand2) = Input.TakeArthmeticInput();
+        if (operand2 == 0)
```
Better use try catch instead of checking operand2 == 0

r-stha replied:
Used DivisionByZeroException in try catch

hstha commented on ArthmeticCalculation.cs:
```diff
@@ -0,0 +1,85 @@
+
+    }
+
+    public static double FactorialRecursion(double number)
```
FactorialRecursion is redundant, you can simply name is Factoral

r-stha replied:
Given the name of factoral

hstha commented on ArthmeticCalculation.cs:
```diff
@@ -0,0 +1,85 @@
+    }
+
+
+    public static double PowerRecursion(double baseNumber, double powerNumber)
```
Please use proper naming, what does PowerRecurssion mean?

hstha commented on Menu.cs:
```diff
@@ -0,0 +1,56 @@
+    public static void ArthmeticCalculationMenu()
+    {
+        Console.WriteLine(@"1. Addition
+2. Subtraction
```
Use proper formating

hstha commented on Menu.cs:
```diff
@@ -0,0 +1,56 @@
+    public static void ScientificMenu()
+    {
+        Console.WriteLine(@$"1. Trigonometric Functions
+2. Find combination and permutation
```
Use proper formating

hstha commented on Menu.cs:
```diff
@@ -0,0 +1,56 @@
+    public static void DegreeOrRadianMenu()
+    {
+        Console.WriteLine("Do you want x in degree or radian:");
+        Console.WriteLine(@"1. Degree
```
Use proper formating

hstha commented on Menu.cs:
```diff
@@ -0,0 +1,56 @@
+    public static void NumberSystemMenu()
+    {
+        Console.Clear();
+        Console.WriteLine(@"1. Binary to decimal
```
Use proper formating

hstha commented on OptionChoose.cs:
```diff
@@ -0,0 +1,113 @@
+
+    public static void NumberSystemConversionOptionChoose(double userChoice)
+    {
+        if (userChoice == 1)
```
If you are clearing console in both cases than you can take it out them to top

r-stha commented:
changed variable name which starts with Uppercase
Added specific data types instead of using var
Added try catch instead of checking operand
Gave a meaning name to factorial and power calculation
Properly formatted the menu section
Cleared the console for the top
